README.md

```diff
@@ -1645,7 +1650,211 @@ If you find yourself using the `only` modifier often, you can set the [context_b
 
 Components can also access the outer context in their context methods like `get_context_data` by accessing the property `self.outer_context`.
 
-## Rendering JS and CSS dependencies
+## Defining HTML/JS/CSS files
```

Comment:
Documented how to work with the Media class, the inputs, how to override it, etc. So conceptually there's 2 separate steps:
1. "Defining HTML/JS/CSS files" describes how to define/link files to a components
2. "Rendering JS and CSS dependencies" describes how, given that I have components that have linked files, how I render out the `<script>`/`<link>` tags.